ndsl/dsl/dace/orchestration.py

```diff
@@ -463,72 +455,71 @@ def orchestrate(
         dace_compiletime_args = []
 
     if config.is_dace_orchestrated():
-        if hasattr(obj, method_to_orchestrate):
```

Comment:
nothing changed here: just an early return in case of `not hasattr(obj, method_to_orchestrate)`